distance_weight: 1
direction_weight: 0
last_goal_effect: true
last_goal_tolerance: 0.5
